src/App.jsx

```diff
@@ -28,13 +29,26 @@ function App() {
     if (selectedGenders.female && user.gender === "female") return true;
     return false;
    });
-
+   const ascendingEvent =() =>{
+    setIsAscending(!isAscending);
+  };
+  const sortedData = [...filteredData];
+  sortedData.sort((a,b) => {
+    if(isAscending){
+      return a.name.localeCompare(b.name);
+    }
+    else {
```

Comment:
Since `return` statement is present inside `if` condition, this `else` statement is not needed.

You can directly use `return b.name.localeCompare(a.name)`